gamemodes/terrortown/gamemode/shared/sh_main.lua

```diff
@@ -135,6 +136,13 @@ function GM:TTTPlayerColor(model)
 	return COLOR_WHITE
 end
 
+---
+-- @hook
+-- @realm shared
+function GM:Think()
+	UpdateSprint()
```

Comment:
The old sprint system was modular designed, to move it easilyy out. If you (and the other) agree connecting it together, I'm okay with it. Idk why we/I decided creating it modular, but we/I did :D (maybe it got already connected, I just woke up when I saw this line). @saibotk @TimGoll @lebroomer any reasons? (I thought about the option to use the ingame sprinting system of gmod or if there is another system, but this maybe doesn't make much sense)